openfisca_core/scripts/migrations/v16_2_to_v17/xml_to_yaml.py

```diff
@@ -260,7 +266,7 @@ def write_yaml(node, path):
         raise ValueError('Unknown type {}'.format(node_type))
 
 
-def write_legislation(legislation_xml_info_list, path):
+def write_parameters(legislation_xml_info_list, path):
```

Comment:
How is this function used ? It never seems to be called ...

Reply:
It is used by country-specific migration scripts (openfisca-core/openfisca_core/scripts/migrations/v16_2_to_v17/xml_to_yaml_country_template.py for example)

Reply:
Then shouldn't it be changed from `xml_to_yaml.write_legislation(legislation_xml_info_list, target_path)` to ` xml_to_yaml.write_parameters(legislation_xml_info_list, target_path)` in that file ?

Reply:
done